JesusFilm/core
# feat: eng 576 see all button on dropdown

# Description

### Issue

Need to be able to see all languages

[Link to Linear Todo](https://linear.app/jesus-film-project/issue/ENG-576/see-all-button-on-dropdown)

### Solution

Use the [refinements toggleShowMore features](https://www.algolia.com/doc/api-reference/widgets/refinement-list/react/#hook-example)

<img width="807" alt="image" src="https://github.com/user-attachments/assets/53ca347b-d780-4680-ac31-ad81593bbe6b">



# External Changes

Had to change the way we have coded the spacing of the refinement groups

# How to test
Should be able to view a selection of languages
Should be able to search languages
Should be able to click see all button and see all languages

Note on staging there are only 8 different languages available. Prod will have 1000.
